package.json

```diff
@@ -18,16 +18,23 @@
     "express-cache-controller": "^1.1.0",
     "limiter": "^1.1.5",
     "mariadb": "^2.4.1",
-    "memory-cache": "^0.2.0"
-    },
+    "memory-cache": "^0.2.0",
+    "module": "^1.2.5",
+    "sql": "^0.78.0",
+    "typescript": "^3.9.6"
```

Comment:
`typescript`, `sql`, and `module` aren't used. please remove.

Reply:
It doesn't run when I don't have them so I'm going to keep them for now but we can look at it again.

Reply:
Shouldn't need some of these run. `npm uninstall --save sql typescript regenerator-runtime module`